# Small Assignment 4

<em>Due October 28th by 11:59pm PT. Download your notebook with responses as a PDF via HTML and submit to Gradescope.</em>

In Small Assignment 2, we estimated the relationship between housing prices, age of the house, owner occupancy status and the presence of hazardous waste in the neighborhood. Suppose our estimates were given by: 

$\widehat{log(hhvalue)}= 11.45 (3.4) - 0.175 (0.34) * waste - 0.00532  (0.00356)  * nbyrs + 2.68 (1.435) * owner occupied$              

$ N = 560, R^2 = 0.113$
                 
where standard errors are given in parentheses.

As a reminder: **hhvalue** is the housing price measured in dollars and **waste** is the quantity in kgs of hazardous waste in the house's census tract, **nbyrs** is the number of years since the house was built, and **owner_occupied** is a dummy variable for whether the house is occupied by its owner (vs rented).

### 1. Is either the age of house or owner occupancy significant at the 5% level against a 2-sided alternative? Show your work.

In [1]:
0.00532/0.00356

[1] 1.494382

In [2]:
2.68/1.435

[1] 1.867596

step1:Define Hpythesis: 

H0: b3 = 0, b4 = 0

H1: B3 $\neq$0 b4 $\neq$0


step2: calculate t-stat
$$t=\frac{\hat{\beta}_1-0}{se}=\frac{0.00532}{0.00356}= 1.49438202247191$$


step3: Choose significane level and find critical Value

Critical value: 1.96


step4: Reject the null or fail to reject it.
Since |t| value is less than the critical value we fail to reject the null hypothesis.

Step5: Interpret:
This tells us that there is statistical evidence that b3 and b4 are significatnly important.


step1:Define Hpythesis: 

H0: b3 = 0, b4 = 0

H1: B3 $\neq$0 b4 $\neq$0


step2: calculate t-stat
$$t=\frac{\hat{\beta}_1-0}{se}=\frac{2.68}{1.435}= 1.87$$


step3: Choose significane level and find critical Value

Critical value: 1.96


step4: Reject the null or fail to reject it.
Since |t| value is less than the critical value we fail to reject the null hypothesis.

Step5: Interpret:
This tells us that there is statistical evidence that b3 and b4 are significatnly important.


### 2. Dropping _nbyrs_ and _owneroccupied_ from the equation gives the equation below. Are _nbyrs_ or _owneroccupied_ jointly significant at the 5% level? Justify your answer. 

$\widehat{log(hhvalue)}= 11.45 (2.5) - 0.178 (0.36) * waste$              

$ N = 560, R^2 = 0.103$
where standard errors are given in parentheses.

Step 1:

H0 is beta2=0 and beta3=0; 

H1 is beta2 $\neq$0 or beta3 $\neq$ 0

step 2: F stat 3.134

step 3: F table w/ significance .05 and df's 2 and 556: critical value is 3.00


In [2]:
((.113-.103)/2)/((1-.113)/556)

[1] 3.13416

step 4: We would reject because F Test value is greater than critical value. This tells us that nbyrs and owneroccupied isn't signficant (I probably have it backward).

### 3. Does including _nbyrs_ and _owneroccupied_ in the model greatly affect the estimated tradeoff between environmental value and housing prices?

No because the betas didn't evne change at all, so that tell me that there is a great affect from nbyrs and owneroccupied. However, you can look at from the two different tests as well in which it was rejected twice, where the test are testing for different aspects of the equation at .5%.

### 4. Carefully explain the difference between the null hypothesis in your tests in part 1 and part 2. What could you learn by rejecting the null hypothesis in each of them?

This is explaining F and T teset, which are different from one another. In T test we were only testing for "one" variable, where  in F test we were testing for more than one variable and "the stronger hypthesis". 